app/presenters/hyrax/fixity_status_presenter.rb

```diff
@@ -34,7 +34,6 @@ def render_file_set_status
 
       # A weird display, cause we need it to fit in that 180px column on
       # FileSet show, and have no real UI to link to for files/versions :(
-      # rubocop:disable Metrics/MethodLength
```

Comment:
Oh, that was a result of the mixup with overwriting the `.rubocop_todo.yml`.  I'll fix it.